### ISSUE TITLE
Smaller screen size accessibility

### ISSUE THREAD

**Issue by jamshale** [uvicgse/project-virtual-team-2#57]
The projects lack of a scrolling, zoom or media queries servery effects the accessibility when using smaller screen sizes.    

1920 * 1080 resolution @ 150% scale:
![image](https://user-images.githubusercontent.com/31809382/58451708-6b63df80-80c9-11e9-917d-5e74ef465424.png)

Same as above but my-app @ 70% zoom:
$("my-app").css("zoom", "70%")

![image](https://user-images.githubusercontent.com/31809382/58452330-cdbddf80-80cb-11e9-8111-a9fd069961c5.png)

As you can see simple styling can solve this problem. Using media queries that detect problem screen sizes and zoom specific DOM elements could make the application far more accessible for a variety of screen sizes without implementing a scroll bar or disturbing the static nature of the display.

**Comment by sinoyuhanxiao** [comment]
Three ways to solve this:
1. Implementing scroll bars on two sides of the window would be a choice. Changing the `overflow: hidden;` to `overflow: scroll` in `body` can create two scroll bars on the sides. But it probably requires reformatting the style of the content because when scrolling, some redundant margins shows up. 
2. To apply responsive web design, we could add media queries to the end of the CSS file setting `max-width` and `min-width` to detect size changing of window. Then, DOM elements could be resized, hidden, shrinked, enlarged, or moved to make them good-looking on the screen. The idea is simple, it just takes time to redesign the look in different window sizes. I suggest applying both 1 and 2 together. 
3. No media queries needed, adding "ctrl+mouse wheel" function for the app to zoom the screen. In this way, when screen size changes, users can zoom the screen themselves to see the whole content.
